src/views/poi_partial/header.dot

```diff
@@ -1,6 +1,6 @@
 {{
   const ico = IconManager.get(this.poi);
-  const grades = this.poi.blocksByType.grades;
+  const grades = this.poi.blocksByType ? this.poi.blocksByType.grades : null;
```

Comment:
this check should not be necessary as `blocksByType` is now defined in base `Poi` class